client/verta/tests/registry/conftest.py

```diff
@@ -98,12 +98,15 @@ def make_boto_session():                 # No modules in function signature
             return boto3.DEFAULT_SESSION         # 3rd-party module in function body
 
         # 3rd-party modules nested inside type constructs should still be extracted
-        def nested_mulitple_returns_hint(self) -> Union[urllib3.Retry, PIL.UnidentifiedImageError]:
+        def nested_multiple_returns_hint(self) -> Union[urllib3.Retry, PIL.UnidentifiedImageError]:
             return urllib3.Retry or PIL.UnidentifiedImageError
 
         # 3rd-party modules nested inside type constructs should still be extracted
-        @property
         def nested_type_hint(self) -> Type[torch.NoneType]:
             return torch.NoneType
 
     return DependencyTestingModel
+
+
+def make_spacy_error():  # No modules in function signature
+    return spacy.Errors
```

Comment:
Was this pasted here by mistake?

Reply:
My thinking here was to have 3rd party modules called "indirectly" (not sure if that is the correct terminology) from within another function to see if we would catch those in the body (like line 65 of contest.py for example).  I think perhaps we should just rip those out for now, since we are not supporting type hints int eh body for now.

Reply:
I don't quite understand. This function isn't even inside the `class`. There's already a `DependencyTestingModel.make_spacy_error()` defined on line 93. What is this one here on line 111 for?

Reply:
Also as a side note, I don't think we'd catch the use of `spacy.Errors` from the body of `predict()`, but our utilities _would_ catch it when we directly inspect the body of `make_spacy_error()`.

Reply:
Done

Reply:
It actually isn't catching those right now, because the module only exists in the `return` line, and thus does not get returned by `inspect.getclosurevars()`.   I think I may have a solution pending

Reply:
OK, I think I've fixed this.  When a module is aliased (as `pd` in the `make_dataframe()` function of the test class for example), it gets stored inside a different variable when returned by `inspect.getclosurevars()`.  Basically its in`nonlocals` instead of `globals` which is why it wasn't returning in some cases.